tests/e2e/notice_fetcher/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from ted_sws.notice_fetcher.adapters.ted_api import TedAPIAdapter, TedRequestAPI
+from tests.fakes.fake_repository import FakeNoticeRepository
+
+
+@pytest.fixture
+def notice_repository():
```

Comment:
Do not duplicate fixtures. 
The fixture resolution is hierarchical. So remove this one because it is defined in the parent package conftest.py